SPECS/openssl-pkcs11/openssl-pkcs11.spec

```
@@ -1,52 +1,47 @@
-Version: 0.4.10
-Release: 7%{?dist}
-
 # Define the directory where the OpenSSL engines are installed
 %global enginesdir %{_libdir}/engines-1.1
-
-Name:           openssl-pkcs11
 Summary:        A PKCS#11 engine for use with OpenSSL
+Name:           openssl-pkcs11
+Version:        0.4.10
+Release:        8%{?dist}
 # The source code is LGPLv2+ except eng_back.c and eng_parse.c which are BSD
-License:        LGPLv2+ and BSD
+License:        LGPLv2+ AND BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/OpenSC/libp11
 Source0:        https://github.com/OpenSC/libp11/releases/download/libp11-%{version}/libp11-%{version}.tar.gz
-
 Patch0:         openssl-pkcs11-0.4.10-various-bug-fixes.patch
 Patch1:         openssl-pkcs11-0.4.10-search-objects-in-all-matching-tokens.patch
 Patch2:         openssl-pkcs11-0.4.10-add-support-pin-source.patch
 Patch3:         openssl-pkcs11-0.4.10-set-rsa-flag-ext-pkey.patch
 Patch4:         openssl-pkcs11-0.4.10-set-rsa-fips-method-flag.patch
-
-BuildRequires:  autoconf automake libtool
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  libtool
 BuildRequires:  openssl-devel
 BuildRequires:  openssl >= 1.0.2
 BuildRequires:  pkgconfig
 BuildRequires:  pkgconfig(p11-kit-1)
-# Needed for testsuite
-BuildRequires:  softhsm opensc procps-ng
-
-
+%if %{with_check}
 BuildRequires:  doxygen
-
-
-Requires:       p11-kit-trust
+BuildRequires:  procps-ng
+BuildRequires:  opensc
+BuildRequires:  softhsm
 Requires:       openssl-libs >= 1.0.2
+Requires:       p11-kit-trust
+%endif
```

The test dependencies are always in the form of `BuildRequires` because we only run test during build time.

```suggestion
%endif

Requires:       openssl-libs >= 1.0.2
Requires:       p11-kit-trust
```